core/services/workflows/cmd/cre/examples/v2/simple_cron/main.go

```diff
@@ -11,7 +11,7 @@ import (
 func RunSimpleCronWorkflow(runner sdk.DonRunner) {
 	cron := &croncap.Cron{}
 	cfg := &croncap.Config{
-		Schedule: "*/3 * * * * *", // every three seconds
+		Schedule: "*/30 * * * * *", // every 30 seconds
```

Comment:
Maybe some middle ground like 5-10 sec? Otherwise people won't see anything for 30 sec and think that stuff's not working.

Reply:
The real CRON binary doesn't allow any time period less than 30 seconds.